include/app.hrl

```diff
@@ -16,3 +16,5 @@
         {keepalive,true}, % Keep our channel open
         {tos,72}, % Deliver immediately
         {active,false}]). % Retrieve data from socket upon request
+
```

Comment:
Now I see the cfd282b was removeing that type socket_setopt(). But this is a long string of checking an this does not exist at the begining. Git "summarize" the changes and end up the difference is a new line.